README.md

```diff
@@ -1 +1,190 @@
+
+### Usage
+
+  - Feel free to experiment with the project.
+  - To build the project: 
+    >``` npm run build ```
+  - The `index.html` file in `public` folder is the output file that you want to open in the browser.
+  - To add tests or examine different tests that are alreadt available go to:
+    >``` [root_dir]/src/modules/[modulename].test.js ```
+
+### Deployment
+
+You can deploy this project by opening a new github repo and initialize the local git repo and connect it through: ```$ git remote add origin main [your github repo link]``` command.
+Then you can push the project into that repository and if you go to the repository in your account, you'll see the code in the account.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHOR -->
+
+## 👥 Author <a id="authors"></a>
+
+
+👤 **Spencer Okyere**
+
+- GitHub: [@skabeo](https://github.com/skabeo)
+- Twitter: [@black_okyere](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okyere-spencer-9b602623b)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a id="future-features"></a>
+
+- A functional and well documented navlinks.
+- A timestamps for all tasks.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/skabeo/react-todo-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a id="support"></a>
+
+If you like this project consider giving a star to the project and if you want to go furthur make a donation to any one the top 8 charities in Hawaii islands at the link below:
+
+-   [Hawaii Donation](https://www.hawaii-guide.com/the-top-8-charities-you-should-donate-to-in-hawaii)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
+
+- I would like to thank [Ibas Majeed](https://twitter.com/ibaslogic) for putting together a well documented article for the todo-list application.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a id="faq"></a>
+
+- **Is mobile first web design a necessity these days?**
+
+  - Since 2014 mobile web users have surpassed desktop users and the rate of the new users being introduced to web by mobile devices will only increase by time.
+- **Does the project support all platforms?**
+  - Yes! Since it is based on web technology, any device that has a web browser will be able to see the website once it is online.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a id="license"></a>
+
+This project is [MIT](https://github.com/skabeo/react-todo-app/blob/develop/LICENSE) licensed.
```

Comment:
**Broken MIT link!**

- The MIT link is currently broken. You can fix it by creating a MIT.md file, copying the [template](https://choosealicense.com/licenses/mit/), then fill the 'year' and  'full name' spaces with your data.

See the image below.

![MIT](https://user-images.githubusercontent.com/63577080/235629428-a505c76c-7e59-4b5b-a292-6daf2ec15803.png)